landing/docs/examples.md

```
@@ -21,47 +21,47 @@ Practical examples of bc in action for common development scenarios.
 ### Team Structure
 ```bash
 # Product Manager coordinates
-bc spawn pm-01 --role product-manager
+mycel spawn pm-01 --role product-manager
 
 # Manager executes strategy
-bc spawn mgr-01 --role manager --parent pm-01
+mycel spawn mgr-01 --role manager --parent pm-01
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)